src/main/resources/psp_config_file.json

```diff
@@ -6113,7 +6113,7 @@
         "buildingNumber": "13",
         "postalCode": "00161",
         "city": "Padova",
-        "province": "PD"
+        "province": "RM"
```

Comment:
sure ?

Reply:
No.... they reported me only the province but for sure we have to change also the city... :)